W-11646503-audit-log-retention (#229)
mulesoft/docs-access-management: modules/ROOT/pages/audit-log-retention.adoc

= Audit Log Retention Period

> xref:audit-logging.adoc[Audit logs] have a default retention period of six years. Users who have the Audit Log Config Manager permission can configure the retention period for the audit logs in their organization.

Author: ```suggestion
xref:audit-logging.adoc[Audit logs] have a default retention period of six years. Users who have Audit Log Config Manager and Organization Administrator permissions can configure the retention period for the audit logs in their organization.
```

Download your logs periodically if you want to maintain your log files for longer than six years.
To check the audit log retention period for your organization, check the retention period in your root organization's settings, or use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23335/[Audit Logging API^].

When you configure a retention period:

* You can specify a retention period between 30 and 2190 days.
* The date the new retention period takes effect must be a minimum of seven days from the current date.
* Configuring a new retention period affects your organization's compliance policy.

== Before You Begin

Before getting started, ensure that you have an account that has the following permissions:

> * Organization administrator

Reviewer: 1) Move the Audit Log Config Manager permission first and then org admin (see reason below) 
2) Should add this in the parentheses 
Organization administrator (only required if using the UI)

Author: ```suggestion
```

> * Audit log config manager

Author: ```suggestion
* Audit log config manager
* Organization administrator (required only if you are using the user interface to set retention. You do not need this permission if you are using the API.)
```

> 

Author: ```suggestion

Your organization must also have *Try new features* or the Teams feature enabled in Access Management. 

```

== Configure Audit Log Retention Period

include::partial$include-nav-steps-org-admin-audit-log-mgr.adoc[]

. Click the *Settings* tab.
. Next to *Audit Log Retention Period*, click the pencil icon.
. In the *Days* field, enter a value between 30 and 2190.
. In the *Date Effective* field, select the date on which you want the new retention period to start.
. Click *Save Changes*. +
A warning appears, confirming your new retention period and date effective.
+
. Click *Confirm*. +

> The new audit log retention period takes effect on the date you specified.

Reviewer: Should we mention the time when it starts? The UI defaults to 0:0:00 but the API allows specifying a time.

Author: ```suggestion
The new audit log retention period takes effect on the date you specified at 0:0:00 UTC. 
```

= Audit Log Retention Period

xref:audit-logging.adoc[Audit logs] have a default retention period of six years. Users who have Audit Log Config Manager and Organization Administrator permissions can configure the retention period for the audit logs in their organization.

Download your logs periodically if you want to maintain your log files for longer than six years.

> To check the audit log retention period for your organization, check the retention period in your root organization's settings, or use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23335/[Audit Logging API^].

Reviewer 2: I uploaded new docs and so this url is now:
https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23341/

When you configure a retention period:

* You can specify a retention period between 30 and 2190 days.
* The date the new retention period takes effect must be a minimum of seven days from the current date.
* Configuring a new retention period affects your organization's compliance policy.

== Before You Begin

Before getting started, ensure that you have an account that has the following permissions:

* Audit log config manager
* Organization administrator (required only if you are using the user interface to set retention. You do not need this permission if you are using the API.)

> Your organization must also have *Try new features* or the Teams feature enabled in Access Management. 

Reviewer 3: It seems strange to me that "Try new features" is bolded and "Team feature" is not bolded.  Even if *Try new features* ise an item in the GUI, I don't think it should be bolded here because you are not talking about the GUI here. Up to you.

== Configure Audit Log Retention Period

include::partial$include-nav-steps-org-admin-audit-log-mgr.adoc[]

. Click the *Settings* tab.
. Next to *Audit Log Retention Period*, click the pencil icon.
. In the *Days* field, enter a value between 30 and 2190.
. In the *Date Effective* field, select the date on which you want the new retention period to start.
. Click *Save Changes*. +

> A warning appears, confirming your new retention period and date effective.

Reviewer 3: Did you mean to say "effective date" instead of "date effective"?

Reviewer 4: Changed to "effective date" as confirmed by the PM.

+
. Click *Confirm*. +
The new audit log retention period takes effect on the date you specified at 0:0:00 UTC. 

> == See Also

Reviewer 2: For just the Set api (without UI usage), we should call out that org admin role is not needed and only Audit Log Config Manager is sufficient.
The requirement of org admin is only for UI where org admin role is needed along with Audit Log Config Manager.
@rajesha13 fyi so this can be worded correctly.

Reviewer 2: Looks like this is already addressed in line # 19 in this file, so resolving this comment.

* xref:audit-logging.adoc[]

> * https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23335/[Audit Logging API^]

Reviewer 2: This is for set api I think and so this url should be 
https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23343/
